source/isaaclab/isaaclab/envs/mdp/events.py

```diff
@@ -574,7 +574,7 @@ def randomize_joint_parameters(
             distribution=distribution,
         )
         asset.write_joint_friction_coefficient_to_sim(
-            friction_coeff[env_ids[:, None], joint_ids], joint_ids=joint_ids, env_ids=env_ids
+            friction_coeff[env_ids, joint_ids], joint_ids=joint_ids, env_ids=env_ids
```

Comment:
This fix won't work if you have a subset of env ids and joint ids over which the term is operating. There is definitely a bug here if both `env_ids` and `joint_ids` are slice.

The way to fix it is to check if one of them is a slice. If yes, we keep the broadcasting. Otherwise we index directly.